src/resizing.jl

```diff
@@ -292,6 +293,21 @@ function imresize(original::AbstractArray{T,N}, new_size::Dims{N}) where {T,N}
     end
 end
 
+function imresize(original::AbstractArray{T,N}, new_size::Dims{N}, aitp::Interpolations.Lanczos4OpenCV) where {T,N}
+    Tnew = imresize_type(first(original))
+    inds = axes(original)
+    if map(length, inds) == new_size
+        dest = similar(original, Tnew, new_size)
+        if axes(dest) == inds
+            copyto!(dest, original)
+        else
+            copyto!(dest, CartesianIndices(axes(dest)), original, CartesianIndices(inds))
+        end
```

Comment:
I think `OffsetArrays` now preserves axes when you simply do `copy(original)`. Any specific reason for this manual written if branch?

Reply:
Not sure if it is required but it mirrors the 282:295 without the Lanczos interpolation so I just copied this previous method exactly to be safe.